src/recurrent/rnn-time-step.js

```diff
@@ -0,0 +1,169 @@
+import Matrix from './matrix';
+import RandomMatrix from './matrix/random-matrix';
+import Equation from './matrix/equation';
+import RNN from './rnn';
+
+export default class RNNTimeStep extends RNN {
+  constructor(options) {
+    super(options);
+  }
+
+  createInputMatrix() {
+    this.model.input = new RandomMatrix(this.inputSize, 1, 0.08);
+  }
+
+  createOutputMatrix() {
+    let model = this.model;
+    let outputSize = this.outputSize;
+    let lastHiddenSize = this.hiddenSizes[this.hiddenSizes.length - 1];
+
+    //whd
```

Comment:
I did mean to keep this, it reflects the same/similar function from the `RNN` class.  This code will be deprecate in v2, so I think changing it would either require it changed in both locations or to just leave it alone.  This was originally in recurrent.js, which was used as a foundation to eventually get to v2.  I think the less work the better, but for the right reasons, so I think leave it.